suite-native/biometrics/src/components/BiometricOverlay.tsx

```diff
@@ -11,12 +16,74 @@ const overlayWrapperStyle = prepareNativeStyle(utils => ({
     backgroundColor: utils.colors.backgroundTertiaryDefaultOnElevation0,
 }));
 
+const bottomWrapperStyle = prepareNativeStyle(() => ({
+    position: 'absolute',
+    bottom: 40,
+    left: 0,
+    right: 0,
+    alignItems: 'center',
+}));
```

Comment:
Nit: I believe this is redundant and you can apply positioning styles to the `TouchableOpacity`

Reply:
492763e1d2